package.json

```diff
@@ -64,7 +66,7 @@
     "babel-eslint": "^4.1.6",
     "eslint": "^1.10.3",
     "eslint-config-strict": "^7.0.4",
-    "eslint-plugin-filenames": "^0.2.0",
+    "eslint-plugin-filenames": "^1.0.0",
```

Comment:
I was wondering why the test did not pass. I did not modify this line by myself... !!!
Will revert it to previous value. Still struggling my development environment setup properly. I'm currently using docker to have a clean node installation for this work.